app/scripts/modules/ecs/pipeline/stages/disableCluster/ecsDisableClusterStage.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.disableClusterStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'disableCluster',
+      cloudProvider: 'ecs',
+      templateUrl: require('./disableClusterStage.html'),
+      executionConfigSections: ['disableClusterConfig', 'taskStatus'],
```

Comment:
shouldn't need this `executionConfigSections` either

Reply:
Done